impl/src/main/java/org/jboss/weld/xml/XmlSchema.java

```diff
@@ -16,11 +16,14 @@
  */
 package org.jboss.weld.xml;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import javax.enterprise.inject.spi.BeanManager;
 
 public enum XmlSchema {
 
-    CDI10("beans_1_0.xsd", BeanManager.class.getClassLoader()), CDI20("beans_2_0.xsd", BeanManager.class.getClassLoader()), WELD11("weld_1_1.xsd", BeansXmlParser.class.getClassLoader());
+    CDI10("beans_1_0.xsd", BeanManager.class.getClassLoader()), CDI11("beans_1_1.xsd", BeanManager.class.getClassLoader()), CDI20("beans_2_0.xsd", BeanManager.class.getClassLoader()), WELD11("weld_1_1.xsd", BeansXmlParser.class.getClassLoader());
```

Comment:
Hm, all the values use the same CL, maybe we could remove the arg and use `BeanManager.class.getClassLoader()` by default?

Reply:
`WELD11` uses different CL, so I kept it as is.